README.md

```diff
@@ -112,8 +112,8 @@ Read, Write, and Admin for Development Information (branches, commits, and pull
 |**Permission scope**|**Why the app needs it**|
 |---|---|
 |**Read** access to contents & metadata |**Contents** (aka code) and **metadata** are needed to sync development information to Jira. <br><br> **Medadata:** All GitHub apps have read-only metadata permission set by default. This is a [mandatory requirement by GitHub](https://docs.github.com/en/rest/reference/permissions-required-for-github-apps#metadata-permissions) and is needed to provide access to a collection of read-only endpoints with metadata for various resources. These endpoints do not provide sensitive private repository information. Read-only metadata permissions are used for the following webhook: <br> - repository (all events excluding `deleted`) <br><br> **Contents:** Read-only content permissions are used for the following webhooks: <br> - commit comment <br> - create <br> - delete <br> - push <br> - workflow run|
+|**Read** access to actions and deployments|If you want to see build and deployment information in Jira, the app will need read permissions for deployments. This will allow the integration to the webhook `deployment_status` event which occurs when a deployment is created. <br><br> **Deployments:** Read deployment permissions are used for the following webhooks: <br> - deployment status|
```

Comment:
I guess you missed the `listen` after "this will allow the integration to...."

Reply:
Maybe...